source/ssl/include/ssl.ooc

```diff
@@ -0,0 +1,45 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+include openssl/aes, openssl/crypto, openssl/evp, openssl/rand
+
+AesKey: cover from AES_KEY
```

Comment:
I would expect `AES` and `SSL`, since they are acronyms, to be treated differently from names. Thus `AESKey`.